hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -82,4 +123,129 @@
       tests: ["pattgen_stress_all"]
     }
   ]
+
+
+  covergroups: [
+    {
+      name: ctrl_cg
+      desc: '''
+            Covers that all valid enable and polarity settings 
+            for the Pattgen control register have been tested.
+            Individual enable and polarity settings that will be covered include:
+            - Enable pattern generator functionality for Channel 0 (ENABLE_CH0)
+            - Enable pattern generator functionality for Channel 1 (ENABLE_CH1)
+            - Clock (pcl) polarity for Channel 0 (POLARITY_CH0)
+            - Clock (pcl) polarity for Channel 1 (POLARITY_CH1)
+            All valid combinations of the above will also be crossed.
+            '''
+    }
+
+    {
+      name: alert_cg
+      desc: '''
+            Covers fatal fault error alert when forcing command error and recover operation error alert when forcing data error.
```

Comment:
Seems like this exceeds 100 chars / line too. Suggest enabling line length detection in your text editor to flag this.

Reply:
Made each line less that 100 chars

Reply:
Made each line less than 100 chars